src/util/DateUtils.js

```diff
@@ -80,6 +80,36 @@ export default class DateUtils {
     return '';
   }
 
+  /**
+   * Given a string, return a Date object created by parsing it.
+   */
+  static stringToUTCDate(dateString: string): Date {
+    return moment.utc(dateString);
+  }
+
+   /**
+   * Given a string, date format and locale, return a formatted date in the specified locale using
+   * the format specified.
+   * @param dateString date as string
+   * @param format given format for coversion
+   * @param locale given locale
+   * @return a formatted date in the specified locale using
+   * the format specified.
+   * @author Shruti Shetty
+   */
+
+  static formatDateStringToUTC(
```

Comment:
Can you describe what the reason for having this method is as opposed to letting users call
`DateUtils.formatDate(DateUtils.stringtoUTCDate(dateString), format, locale);`
I'm wondering how commonly used this would be... if it's just for the DocumentEntityList component, maybe it doesn't warrant being a separate method?